src/physics/PythiaGenerator.cc

```diff
@@ -131,7 +131,7 @@ Particle _convert_particle(Pythia8::Particle& particle) {
                particle.tProd() * mm / c_light,
                particle.zProd() * mm,
                particle.yProd() * mm,
-              -particle.xProd() * mm + 81*m + static_cast<double>(Earth::LastShift())};
+              -particle.xProd() * mm + static_cast<double>(Cavern::IP())};
   out.set_pseudo_lorentz_triplet(particle.pT() * GeVperC, particle.eta(), particle.phi() * rad);
```

Comment:
Do we take into account that the LHC is not horizontal at the IP? That is, the eta = 0 plane is rotated 0.704 degrees from the global z-axis.

Reply:
We don't take that into account in the simulation. I don't think we care about that level of accuracy in our simulation. We can talk to Gordon and JP about this.